src/time_tracker/invoices/handlers/spec.clj

```diff
@@ -1,14 +1,50 @@
 (ns time-tracker.invoices.handlers.spec
   (:require [clojure.spec :as s]
-            [time-tracker.spec :as core-spec]))
-
-(s/def ::start ::core-spec/epoch)
-(s/def ::end ::core-spec/epoch)
-;; A non-empty string.
-(s/def ::client (s/and string?
-                       seq))
-
-(s/def ::generate-invoice-params
-  (s/and (s/keys :req-un [::start ::end ::client])
-         (fn [{:keys [start end]}]
-           (< start end))))
+            [time-tracker.spec :as core-spec]
+            [time-tracker.invoices.spec :as invoices-spec]
+            [time-tracker.users.spec :as users-spec]
+            [time-tracker.timers.spec :as timers-spec]))
+
+(s/def ::rate number?)
+(s/def ::user-id->rate-map-raw (s/keys :req-un [::invoices-spec/user-id ::rate]))
```

Comment:
Similar to https://github.com/nilenso/time-tracker/pull/63/files#diff-9960a0f13bfce1a10c44230260f8a27eR13